bridge/cypress/support/pageobjects/SequencesPage.ts

```diff
@@ -9,6 +9,18 @@ export class SequencesPage {
     return this;
   }
 
+  public assertTaskState(taskName: string, isFailed: boolean, isSuccess: boolean): this {
```

Comment:
I think we should split this into `assertTaskFailed` and `assertTaskSuccessful` as it is either one, or the other, or none of both. If you return `this` you can chain them anyways.
And when split It is independent to use.